stress/common/src/main/java/alluxio/stress/client/Operation.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.client;
+
+/**
+ * The operations for the master stress tests.
+ */
+public enum Operation {
```

Comment:
would like this to be ClientOperation (and maybe MasterOperation on the other operation as well)

Reply:
renamed this one.